Docs/source/usage/parameters.rst

```diff
@@ -2088,6 +2088,13 @@ Details about the collision models can be found in the :ref:`theory section <mul
     a Coulomb logarithm will be computed automatically according to the algorithm in
     :cite:t:`param-PerezPOP2012`.
 
+* ``<collision_name>.use_global_debye_length`` (`bool`) optional
+    Only for ``pairwisecoulomb``. When set, the Debye length used in the Coulomb log
+    is calculated including all species in the simulation. The lengths are combined
+    using the square root of one over the sum of one over the sqaures of the Debye lengths
+    of each species. By default, this is turned off. Note that if ``<collision_name>.CoulombLog``
+    is specified, this Debye length is not used.
```

Comment:
Wouldn't it be easier to write a formula (or pseudo-formula) rather than explaining in words as `The lengths are combined using the square root of one over the sum of one over the sqaures of the Debye lengths of each species`? If you prefer the word explanation, note the typo `sqaures`.

Reply:
In this case, I prefer the written description - it seems more concise. Writing a formula would take up more space and would require the extra text defining terms and subscripts.